### DISCUSSION TITLE
ДЛЯ РАЗГОВОРОВ О ДОСТУПНОСТИ ПРОДУКТОВ GITHUB

### DISCUSSION THREAD

**Discussion by Suleyman313** [community/community#113589] [Accessibility]
### Select Topic Area

Product Feedback

### Body

GitHub и управление торговлей
В этой статье
[Обзор экспорта](https://docs.github.com/ru/site-policy/other-site-policies/github-and-trade-controls#export-overview)
[Вопросы и ответы](https://docs.github.com/ru/site-policy/other-site-policies/github-and-trade-controls#frequently-asked-questions)
GitHub.com, GitHub Enterprise Server и информация, которую вы загружаете в любой продукт, могут подпадать под действие правил торгового контроля, в том числе в соответствии с Правилами экспортного контроля США (EAR).

Видение GitHub — стать глобальной платформой для совместной работы разработчиков, независимо от того, где они находятся. Мы серьезно относимся к нашей обязанности тщательно изучить правительственные санкции, чтобы убедиться, что пользователи и клиенты не пострадали сверх того, что требуется по закону. Это включает в себя обеспечение наличия и доступности услуг общедоступных репозиториев, в том числе для проектов с открытым исходным кодом, для поддержки личного общения с участием разработчиков в регионах, на которые распространяются санкции. Это также означает, что GitHub будет выступать за разработчиков в санкционированных регионах, чтобы они получили более широкий доступ к платформе и полный доступ к глобальному сообществу открытого исходного кода.

Благодаря нашей поддержке и напряженной работе с регулирующими органами США GitHub получил лицензию от Управления по контролю за иностранными активами Министерства финансов США (OFAC) на восстановление наших облачных сервисов для разработчиков в Иране. И мы продолжим работать с регулирующими органами США, чтобы получить аналогичные лицензии для предоставления услуг совместной работы разработчикам в других регионах, на которые распространяются санкции, таких как Сирия и Крым. Мы стремимся выступать за максимально широкий доступ для разработчиков, поскольку считаем, что предоставление услуг совместной работы над кодом для разработчиков в регионах, на которые распространяются санкции, делает глобальное сообщество разработчиков сильнее, способствует человеческому прогрессу и поддерживает неизменную внешнюю политику США по продвижению свободы слова и свободного потока. информации. Сведения о лицензии, предоставляемой разработчикам из Ирана, см. [в записи блога](https://github.blog/2021-01-05-advancing-developer-freedom-github-is-fully-available-in-iran).

Хотя мы предоставили следующую информацию ниже для вашего удобства, в конечном итоге вы несете ответственность за обеспечение того, чтобы использование вами продуктов и услуг GitHub соответствовало всем применимым законам и нормам, включая законы США о контроле за экспортом.

[Обзор экспорта](https://docs.github.com/ru/site-policy/other-site-policies/github-and-trade-controls#export-overview)
[GitHub.com](https://docs.github.com/ru/site-policy/other-site-policies/github-and-trade-controls#githubcom)
В разделе [Условия обслуживания](https://docs.github.com/ru/site-policy/github-terms/github-terms-of-service) пользователи могут получать доступ и использовать GitHub.com только в соответствии с действующим законодательством, включая законы США о контроле за экспортом и санкциях.

Пользователи несут ответственность за обеспечение того, чтобы контент, который они разрабатывают и публикуют на GitHub.com, соответствовал законам США об экспортном контроле, включая EAR и Правила международной торговли оружием США (ITAR). Предложение облачной службы доступно по адресу [GitHub.com](https://github.com/), не предназначено для размещения данных, подпадающих под действие ITAR, и в настоящее время не предлагает возможности ограничения доступа к репозиторию по стране. Если вы хотите сотрудничать с ITAR или другими данными, контролируемыми экспортом, мы рекомендуем вам рассмотреть продукт [GitHub Enterprise Server](https://enterprise.github.com/), локальное предложение GitHub.

GitHub теперь имеет лицензию OFAC на предоставление облачных услуг разработчикам, находящимся или иным образом проживающим в Иране. Сюда входят все государственные и частные услуги для частных лиц и организаций, как бесплатные, так и платные.

Облачные сервисы GitHub, как бесплатные, так и платные, также обычно доступны для разработчиков, находящихся на Кубе.

Гражданам особых категорий (SDN), другим запрещенным или заблокированным сторонам в соответствии с законодательством США и другими применимыми законами, а также некоторым государственным служащим может быть ограничен доступ или использование GitHub.com. Кроме того, пользователи не могут использовать GitHub.com для или от имени таких сторон, как правило, включая правительства стран, на которые распространяются санкции. Кроме того, GitHub.com нельзя использовать в целях, запрещенных в соответствии с применимыми законами о контроле экспорта, включая запрещенные конечные виды использования, описанные в [15 CFR 744](https://www.ecfr.gov/cgi-bin/text-idx?SID=ad384e1f1e017076f8c0136f322f0a4c&mc=true&node=pt15.2.744&rgn=div5).

[GitHub Enterprise Server](https://docs.github.com/ru/site-policy/other-site-policies/github-and-trade-controls#github-enterprise-server)
GitHub Enterprise Server — это автономное виртуальное устройство, которое можно запустить в вашем собственном центре обработки данных или в виртуальном частном облаке. Таким образом, GitHub Enterprise Server можно использовать для хранения ITAR или другой информации, контролируемой экспортом, хотя конечные пользователи, тем не менее, несут ответственность за обеспечение соблюдения ITAR и других применимых мер контроля экспорта.

GitHub Enterprise Server — это коммерческий продукт для массового рынка, которому присвоен классификационный номер экспортного контроля (ECCN) 5D992.c и который может быть экспортирован в большинство стран без лицензии (NLR).

GitHub Enterprise Server нельзя продавать, экспортировать или реэкспортировать в любую страну, указанную в группе стран E:1 в Приложении №. 1 к части 740 EAR (правила экспортного управления США) или в следующих регионах Украины: Крым и сепаратистские районы Донецка и Луганска. Перечисленные страны в настоящее время включают Кубу, Иран, Северную Корею, Сирию, Россию и Беларусь, но этот список может быть изменен.

[GitHub Copilot](https://docs.github.com/ru/site-policy/other-site-policies/github-and-trade-controls#github-copilot)
GitHub Copilot — это коммерческий продукт для массового рынка, которому присвоен классификационный номер экспортного контроля (ECCN) 5D992.c и который может быть экспортирован в большинство стран без лицензии (NLR).

GitHub Copilot нельзя продавать, экспортировать или реэкспортировать в страны, на которые наложено эмбарго, и страны, указанные в группе стран E:1 в Приложении №. От 1 до 740 части EAR (правил экспортного управления США) без авторизации. В настоящее время к таким назначениям относятся Куба, Иран, Северная Корея, Сирия, Россия, Беларусь и следующие регионы Украины: Крым/Севастополь и сепаратистские районы Донецк и Луганск. Этот список может быть изменен.

[Вопросы и ответы](https://docs.github.com/ru/site-policy/other-site-policies/github-and-trade-controls#frequently-asked-questions)
[В отношении каких стран и территорий применяются санкции правительства США?](https://docs.github.com/ru/site-policy/other-site-policies/github-and-trade-controls#on-which-countries-and-territories-are-us-government-sanctions-applied)
Крым, сепаратистские районы Донецка и Луганска, Куба, Иран, Северная Корея и Сирия. Однако, что касается Ирана, GitHub теперь имеет лицензию Управления по контролю за иностранными активами Министерства финансов США (OFAC) на предоставление облачных услуг разработчикам, находящимся или иным образом проживающим в этой стране. Облачные сервисы GitHub, как бесплатные, так и платные, также обычно доступны для разработчиков, находящихся на Кубе.

**Comment by ghostinhershell** [comment]
Hey there!  👋

Thanks for posting in the GitHub Community, @Suleyman313! We're happy you're here. You are more likely to get a useful response if you are posting your question in the applicable category. The Accessibility category is a place for our community to discuss and provide feedback on the digital accessibility of GitHub products. Digital accessibility means that GitHub tools, and technologies, are designed and developed so that people with disabilities can use them.

I've gone ahead and moved this to the correct category for you. Good luck!
